external-crates/move/crates/move-trace-format/src/format.rs

```diff
@@ -217,6 +217,12 @@ impl MoveTrace {
     }
 }
 
+impl Default for MoveTrace {
```

Comment:
Oh clippy...

<img src="https://media4.giphy.com/media/qsnHYz7lXfDokIHfrd/giphy.gif?cid=bd3ea57eaj2i1s9wieoucxt1edlpxbcqnhi1dvszt5t7nft4&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>

Reply:
We have this clippy note disabled in `external-crates` fwiw. So we can remove this if we want

Reply:
> We have this clippy note disabled in `external-crates` fwiw. So we can remove this if we want

Fwiw, I got clippy complaining about it in CI...